web/styles/zulip.css

```diff
@@ -1286,6 +1294,10 @@ div.focused-message-list {
     font-weight: bold;
 }
 
+.stream-topic {
+    white-space: pre-wrap;
+}
```

Comment:
Where is this class used? I see a `stream_topic` class used in a few places, but not this one.

Reply:
`.stream-topic` is attached to topic links inside the markdown based messages. The notification bot's message containing link to a topic is one example in which backend automatically adds the message moved notification with this class.

Reply:
I've checked but couldn't find any other uses for this class, therefore added pre here.

Reply:
OK, in that case it should be added to `rendered_markdown.css` as `.rendered_markdown .stream_topic`, so it's scoped to just that use case.